### PR TITLE
Add 2022q3 OpenStack on FreeBSD project update

### PR DESCRIPTION
**Pull request by starbops**

### CODE REVIEW

File: 2022q3/openstack-on-freebsd.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Links: +
+link:https://www.openstack.org/[OpenStack] URL: link:https://www.openstack.org/[] +
+link:https://github.com/starbops/osf[OpenStack on FreeBSD] URL: link:https://github.com/starbops/osf[] +
+
+Contact: Chih-Hsin Chang <starbops@hey.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+OpenStack is an open-source cloud operating system for different types of resources like virtual machines and bare metals.
+However, OpenStack only unofficially supports FreeBSD as a guest system.
```
Comment (pauamma) [inline 2022q3/openstack-on-freebsd.adoc:11]: Meaning of "unofficially" unclear. "only supports FreeBSD as a guest, and that support isn't official"? Or is it "only unofficially" as in "no official support"? Either way, rhe next sentence about FreeBSD as a host sounds like a non-sequitur. Can you explain?

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q3/openstack-on-freebsd.adoc (hunk 1 above)
Comment (starbops) [reply 2022q3/openstack-on-freebsd.adoc:11]: I think the sentence is not accurate. According to the [docs page](https://docs.openstack.org/image-guide/obtain-images.html#freebsd-openbsd-and-netbsd), it is the FreeBSD guest image is unofficially maintained, not the support of OpenStack is unofficial. OpenStack only maintains a [support matrix regarding the hypervisors](https://wiki.openstack.org/wiki/HypervisorSupportMatrix). To make the intent of the paragraph clear, I'll just remove the line:

> However, OpenStack only unofficially supports FreeBSD as a guest system.

What do you think?

File: 2022q3/openstack-on-freebsd.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* Neutron (networking service)
+* Nova (compute service)
+
+Some of the items are still under heavy development.
+For instance, due to the design of Neutron, the DHCP servers are placed inside Linux network namespaces.
+It is necessary to find an alternative, e.g. `vnet`, on FreeBSD and adapt it.
+Most of the time the porting strategy is to make as small of an impact as possible by working around obstacles.
+But something like `oslo.privsep` deserves a true porting. `oslo.privsep` is rooted in Linux capabilities to do the privilege separation work.
+Right now we just bypassed any Linux capabilities-related operation inside `oslo.privsep`.
+So there are plenty of hackish stuff in the source code and configurations currently.
```
[the remark this answers is not in the document]
Comment (starbops) [reply 2022q3/openstack-on-freebsd.adoc]: Updated, thank you.

File: 2022q3/openstack-on-freebsd.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
+=== OpenStack on FreeBSD
+
+Links: +
+link:https://www.openstack.org/[OpenStack] URL: link:https://www.openstack.org/[] +
+link:https://github.com/starbops/osf[OpenStack on FreeBSD] URL: link:https://github.com/starbops/osf[] +
+
+Contact: Chih-Hsin Chang <starbops@hey.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+OpenStack is an open-source cloud operating system for different types of resources like virtual machines and bare metals.
```
[the remark this answers is not in the document]
Comment (starbops) [reply 2022q3/openstack-on-freebsd.adoc]: Updated, thank you.

**Review by Ged16 — COMMENTED** [review]
Request approval

File: 2022q3/openstack-on-freebsd.adoc (hunk 1 above)
Comment (pauamma) [reply 2022q3/openstack-on-freebsd.adoc]: Removing it works for me.

**Review by pauamma — APPROVED** [review]
First 2 changes grahamperrin requested taken care of. I'll fix capitalization when I merge.

**Pull request merged**